base/ltmeta.dtx

```diff
@@ -122,11 +122,12 @@
 %    \end{macrocode}
 %
 %    \begin{macrocode}
-%<latexrelease>\NewModuleRelease{2021/06/01}{ltmeta}
+%<latexrelease>\NewModuleRelease{2022/06/01}{ltmeta}
 %<latexrelease>                 {Document Metadata handling}
```

Comment:
guess that's wrong. This makes the  module appear only in 2022/06 but it was added in 2021.

Reply:
the first commit was in dezember 2021, and \DocumentMetadata is currently defined only in latex-dev, so that makes it 2022 doesn't it?